lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -487,10 +489,16 @@ def ensure_ok(self):
             # Zones and listeners will be added at creation
             self._create_elb()
         else:
-            self._set_zones()
-            self._set_security_groups()
-            self._set_elb_listeners()
-            self._set_subnets()
+            new_scheme = self._get_scheme()
```

Comment:
This is only used on the line immediately below, so just use it there instead of assigning.